### PR TITLE
Issue#39 add info to report

### PR DESCRIPTION
**Pull request by ahaselsteiner**

### CODE REVIEW

File: enviro/plot.py [hunk 1]
```diff
@@ -418,54 +424,215 @@ def define_header_and_footer(canvas, doc):
...
+    if reached_max_eedc_number:
+        table_string += "Only the first " + str(MAX_EEDCS_TO_LIST_IN_TABLE) + " out of " + str(len(matrix[0][1])) + " EEDCs are listed."
+
+    return table_string
+
+def get_latex_eedc_table_head_line(var_names):
+    """
+        Creates a latex string containing a the first line of a table.
+
+        The table lists the contour's extreme environmental conditions.
```
Comment (kawindme) [inline enviro/plot.py]: "extreme environmental design conditions"

File: enviro/plot.py [hunk 2]
```diff
@@ -418,54 +424,215 @@ def define_header_and_footer(canvas, doc):
...
+
+    Makes use of the 'latex_report.tex' template where the document class and packages are defined.
+
+    Parameters
+    ----------
+    matrix : n-dimensional matrix
+        The coordinates of the environmental contour.
+        The format is defined by compute_interface.iform()
+
+    user : Django User instance
```
Comment (kawindme) [inline enviro/plot.py]: user.models.User

File: enviro/plot.py [hunk 3]
```diff
@@ -418,54 +424,215 @@ def define_header_and_footer(canvas, doc):
...
+
+    method_label : string
+        Will be used as the title of the contour plot,
+        e.g.  "Tom's wave model, T = 2 years, Highest Density Contour (HDC)"
+
+    probabilistic_model : ProbabilisticModel instance
```
Comment (kawindme) [inline enviro/plot.py]: enviro.models.ProbabilisticModel

File: enviro/plot.py [hunk 4]
```diff
@@ -418,54 +424,215 @@ def define_header_and_footer(canvas, doc):
...
+
+    var_names : list of strings
+        Names of the environmental variables used in the probabilistic model,
+        e.g. ['wind speed [m/s]', 'significant wave height [m]']
+
+    var_symbols : list of strings
+        Symbols of the environental variables used in the probabilistic model,
+        e.g. ['V', 'Hs']
+
+    method : views.Method instance
```
Comment (kawindme) [inline enviro/plot.py]: enviro.views.Method

File: enviro/plot.py [hunk 5]
```diff
@@ -17,9 +17,15 @@
...
-from pprint import pprint
+from django.http import HttpResponse
+from django.template import Context
+from django.template.loader import get_template
+from subprocess import Popen, PIPE
+from .compute_interface import setup_mul_dist
+import tempfile
+
+IMAGE_FULL_WIDTH_REPORT = 16.26  # in cm
+IMAGE_FULL_HEIGHT_REPORT = 12.19  # in cm
```
Comment (kawindme) [inline enviro/plot.py]: IMAGE_FULL_WIDTH_REPORT and IMAGE_FULL_HEIGHT_REPORT seem to be unused

File: enviro/plot.py (hunk 2 above)
Comment (ahaselsteiner) [reply enviro/plot.py]: I think this should be "user : django.contrib.auth.models.User"

**Comment by ahaselsteiner** [comment]
Thanks Kai. I addressed the comments you had so far.

**Pull request merged**